src/components/Select/components/SelectInner.vue

```diff
@@ -91,7 +91,7 @@
     <div class="sb-select-inner__icons">
       <button
         v-if="showClearButton && !isDisabled"
-        v-tooltip="{ label: 'Remove all' }"
+        v-tooltip="{ label: 'Clear selection' }"
```

Comment:
I changed this so it fits better to what this button really does, and "selection" fits better than "all" for single and multiple select